CLAUDE.md

```
@@ -137,35 +141,262 @@ If you prefer direct commands without justfile or the modern CLI:
+       return asyncio.run(scrape_async(url))
+   ```
+
+3. **Use asyncio.gather() for concurrent operations**:
+
+   ```python
+   results = await asyncio.gather(*tasks, return_exceptions=True)
+   ```
+
+### Rust Optimization Patterns
+
+1. **Use Cow<str> for zero-copy strings**:
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: str

(MD033, no-inline-html)